cypress/e2e/tests/migration/task-manager/task_manager.test.ts

```diff
@@ -79,6 +81,14 @@ describe(["@tier1"], "Task Manager", () => {
         clearAllFilters();
     });
 
+    it("Delete an application - related tasks are deleted", function () {
+        // Remove the last element from and return it
```

Comment:
One minor issue : 
Did you mean to add  'Remove the last element from application list?' ?

Reply:
Yes. Thanks for pointing this out!

Reply:
Done in https://github.com/konveyor/tackle-ui-tests/pull/1288/commits/fbe9781303258f98146831aeaddfca8fb6d4ccd3